### DISCUSSION TITLE
uYouPlus.com

### DISCUSSION THREAD

**Discussion by SkyJoshua** [qnblackcat/uYouPlus#1373] [General]
Is uYouPlus.com an official website of this or its it likely a virus?

**Comment by yodaluca23** [comment]
> Is uYouPlus.com an official website of this or its it likely a virus?

I don't think it's owned by any of the maintainers, but all the links match up, and it has correct information, it's probably made by a fan. According to the Whois database, the owner is located in AP, India.

**Reply by SkyJoshua** [reply]
okay thank you!
